benchmarks/dynamo/common.py

```diff
@@ -1098,14 +1098,27 @@ def maybe_mark_profile(*args, **kwargs):
     torch._dynamo.config.repro_tolerance = tolerance
 
     with maybe_profile(args.export_profiler_trace, **args.profile_details) as p:
-        if args.export_aot_inductor:
-            frozen_model_iter_fn = export_aot_inductor(
-                model, example_inputs, args.inductor_compile_mode
-            )
-        elif args.export_nativert:
-            frozen_model_iter_fn = export_nativert(model, example_inputs)
+        use_generate_mode = kwargs.get("use_generate_mode", False)
+        if use_generate_mode:
```

Comment:
I added this use_generate_mode flag so that we only apply torch.compile/export to model.forward, instead of applying it to model.generate